### PR TITLE
Feature/environment revision

### PR DESCRIPTION
**Pull request by erikwithuhk**
I reorganized the environment structure to mimic the app Andrew shared yesterday, so I moved the React components into client, back-end stuff into app, etc.

Next step will be to set up the server so that I can see if this thing runs correctly, and if not, I will debug.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
     "babel-loader": "^6.2.5",
     "babel-preset-es2015": "^6.16.0",
     "babel-preset-react": "^6.16.0",
+    "eslint-plugin-import": "^2.0.0",
+    "express": "^4.14.0",
+    "jasmine": "^2.5.2",
```
Comment (LynnFleck) [inline package.json:27]: I don't think we'll be using Jasmine for this project

File: package.json [hunk 2]
```diff
@@ -35,6 +38,7 @@
     "eslint-config-airbnb": "^12.0.0",
     "eslint-plugin-import": "^2.0.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
-    "eslint-plugin-react": "^6.3.0"
+    "eslint-plugin-react": "^6.3.0",
+    "jasmine": "^2.5.2"
```
Comment (LynnFleck) [inline package.json:42]: we won't need Jasmine

File: package.json [hunk 3]
```diff
@@ -4,8 +4,8 @@
   "description": "## Problem Statement:",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dev": "webpack-dev-server --hot --progress --colors"
+    "test": "jasmine",
```
Comment (LynnFleck) [inline package.json:7]: don't need jasmine

**Review by LynnFleck — APPROVED** [review]
my only comments were re: jasmine, so I'm marking this as approved

File: package.json (hunk 2 above)
Comment (erikwithuhk) [reply package.json:42]: Yeah, I threw it in there in case we eventually got around to testing, but I really doubt we'll have time.

**Pull request merged**